MANIFEST.in

```diff
@@ -0,0 +1 @@
+include src/schnetpack/train/ressources/partition_spline_for_robust_loss.npz
```

Comment:
What is this?

Reply:
From this file values are loaded which are used to approximate the partition function to for the adaptive loss fn